sycl/include/sycl/detail/pi.h

```diff
@@ -654,6 +654,8 @@ constexpr pi_queue_properties PI_QUEUE_FLAG_ON_DEVICE_DEFAULT = (1 << 3);
 constexpr pi_queue_properties PI_EXT_ONEAPI_QUEUE_FLAG_DISCARD_EVENTS = (1 << 4);
 constexpr pi_queue_properties PI_EXT_ONEAPI_QUEUE_FLAG_PRIORITY_LOW = (1 << 5);
 constexpr pi_queue_properties PI_EXT_ONEAPI_QUEUE_FLAG_PRIORITY_HIGH = (1 << 6);
+constexpr pi_queue_properties PI_EXT_ONEAPI_QUEUE_FLAG_BATCHED_SUBMISSION = (1 << 7);
+constexpr pi_queue_properties PI_EXT_ONEAPI_QUEUE_FLAG_IMMEDIATE_SUBMISSION = (1 << 8);
```

Comment:
As this PR changes `pi.h` then it will require UR changes to match this.

As stated in a comment on another PR, documentation on how to add experimental features is being written here https://github.com/oneapi-src/unified-runtime/pull/546

But this change may be small enough and concrete enough to skip experimental and we just add it to UR directly?

You can ask us (@alycm42, @jandres742, @kbenzie) any questions and we'll help you out.